lib/cost_calculator.rb

```diff
@@ -0,0 +1,23 @@
+class CostCalculator
+
+  def calculate(current_order, menu_contents)
+    cost_of_order = 0
+    current_order.each do |index|
+      cost_of_order += price_for_quantity(index, menu_contents)
+    end
+    cost_of_order
+  end
+
```

Comment:
All methods below feel like they should be private. 
As a side note, you should not test private methods.